src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildAssetsTest.cs

```diff
@@ -58,9 +58,14 @@ public void CheckPostCompileAssetsIncludedInAPK ()
 		}
 
 		[Test]
-		public void CheckAssetsAreIncludedInAPK ()
+		public void CheckAssetsAreIncludedInAPK ([Values (true, false)] bool useAapt2)
 		{
-			var projectPath = string.Format ("temp/CheckAssetsAreIncludedInAPK");
+			if (IsWindows && useAapt2) {
+				//NOTE: currently on Windows aapt2 produces a `assets\subfolder/asset2.txt` path in `packaged_resources`
+				Assert.Ignore ("This test is not currently working on Windows.");
+			}
```

Comment:
I'm not entirely sure this is the best course of action.

What do we do when aapt2 is broken on Windows?